### PR TITLE
Update readme, and version tags for services.

### PR DESCRIPTION
**Pull request by nzcodarnoc**
As pull requests are approved, the branch is deleted. So I tag the merge so that docker-compose keeps having the same code in `context:`

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ services:
     image: korerorero-front-end:latest
     container_name: korerorero-front-end
     build:
-      context: https://github.com/ServiceInnovationLab/korerorero-front-end.git#install-chat-ui
+      context: https://github.com/ServiceInnovationLab/korerorero-front-end.git#v0.1.0
```
Comment (JacOng17) [inline docker-compose.yml:7]:
```suggestion
      context: https://github.com/ServiceInnovationLab/korerorero-front-end.git
```

Comment (JacOng17) [reply docker-compose.yml:7]: image

**Review by JacOng17 — CHANGES_REQUESTED** [review]
LGTM, other than the comment that I suggested. :+1:

File: docker-compose.yml (hunk 1 above)
Comment (nzcodarnoc) [reply docker-compose.yml:7]: I forgot to push the `v0.1.0` tag. Try it again with the tag?

Comment (JacOng17) [reply docker-compose.yml:7]: :+1:

**Review by JacOng17 — APPROVED** [review]
LGTM, works! :+1:

**Pull request merged**